File: BookAlleyWebApi/Models/Post.cs
namespace BookAlleyWebApi.Models
{
    public class Post
    {
        public long Id { get; set; }
        public required User Poster { get; set; }
        public string? Author { get; set; }
        public string? ISBN { get; set; }
        public required string BookTitle { get; set; }
        public string? Image { get; set; }
        public required string Location { get; set; }
        public string? Note { get; set;}
        public required DateTimeOffset DatePosted { get; set; }
        public bool? IsDeleted { get; set; }
    }
}

File: BookAlleyAndroid/app/src/main/java/com/example/bookalleyandroid/Models/Post.java
package com.example.bookalleyandroid.Models;

import java.time.OffsetDateTime;

public class Post {
    public long Id;
    public String PosterName;
    public long PosterId;
    public String BookTitle;
    public String Author;
    public String ISBN;
    public String Image;
    public String Location;
    public String Note;
    public OffsetDateTime DatePosted;
    public int Quantity;
}
